ext/standaloneusers/Civi/Api4/Session.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Civi\Api4;
+
+/**
+ * Session entity.
+ *
+ * Provided by the Standalone Users extension.
+ *
+ * @package Civi\Api4
+ */
+class Session extends Generic\DAOEntity {
```

Comment:
I wonder if it's appropriate to expose the `civicrm_session` table as an api? If it's not needed now, maybe we should leave this out until there's a use-case for it.
Skimming this PR, it appears to be only used in a test, and that test could probably be switched to use BAO methods instead.

Reply:
Ok, makes sense. In that case we probably need to add a `permissions()` function to this file which restricts access to only super-admins.

Reply:
OK, I'll do that 👍🏻